File: src/JavaScript/easy/string/problem2810.js
/**
 * @param {string} s
 * @return {string}
 */
var finalString = function(s) {
    let finalStr = []
    s.split("").forEach(str => {
        str == "i" ? finalStr.reverse() : finalStr.push(str)
    })
    return finalStr.join("")
};

// let s = "string"
let  s = "poiinter"

finalString(s)

File: src/Python/easy/string/problem2810.py
class Solution:
    def finalString(self, s: str) -> str:
        finalStr = []
        for str in s:
           finalStr.reverse() if str == "i" else finalStr.append(str)
        return "".join(finalStr)


s = "string"

solution = Solution()
solution.finalString(s)